src/main/java/roomescape/controller/api/MemberApiController.java

```diff
@@ -21,10 +21,11 @@ public MemberApiController(MemberService memberService) {
         this.memberService = memberService;
     }
 
-    @PostMapping("/members")
-    public ResponseEntity<MemberIdAndNameResponse> signup(@RequestBody @Valid SignupRequest request) {
+    @PostMapping("/api/members")
+    public ResponseEntity<MemberIdAndNameResponse> signup(@RequestBody @Valid
+                                                          SignupRequest request) {
```

Comment:
```suggestion
    public ResponseEntity<MemberIdAndNameResponse> signup(
           @RequestBody @Valid SignupRequest request
    ) {
```
파라미터는 어노테이션을 개행하면 어디에 적용하는지 쉽게 안보여서 한줄로 처리할 수 있도록 해주는게 좋을것 같아요!